packages/core/src/api/composed/spore/meltThenCreateSpore.ts

```diff
@@ -56,6 +58,24 @@ export async function meltThenCreateSpore(props: {
     cellProvider: indexer,
   });
 
+  // Insert input cells in advance for particular purpose
+  if (props.fromCells) {
+    txSkeleton.update('inputs', (inputs) => {
+      for (const cell of props.fromCells!) {
+        const address = encodeToAddress(cell.cellOutput.lock, { config: config.lumos });
```

Comment:
Why is an `address` variable being created here? This `address` is not used later on.

Reply:
> Why is an `address` variable being created here? This `address` is not used later on.

no, it's not useless, this `address` will be pushed into `fromInfos` if no same item in it later on